integration-tests/tasks/e2e-test.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: e2e-test
+spec:
+  description: >-
+    This task can be used to run e2e tests
+  params:
+    - name: e2e_test_namespace
+      type: string
+    - name: app_suffix
+      type: string
+    - name: ec_pipelines_repo_url
+      type: string
+    - name: ec_pipelines_repo_revision
+      type: string
+    - name: custom_source_build_bundle
+      type: string
+  steps:
+    - name: e2e-test
+      image: quay.io/susdas/e2e:next
+      # a la infra-deployment updates, when PRs merge in e2e-tests, PRs will be opened
+      # against build-definitions to update this tag
```

Comment:
Why "against build-definitions"?

Reply:
the task file I copied from the e2e-test task definitions that we run CI on build-definitions repo, didn't notice it, will remove the comments.